README.md

```diff
@@ -1,5 +1,33 @@
 **⚠️ UNDER CONSTRUCTION ⚠️**
 
+Make an order through API CowSwap
+
+Dependencies
+python3 version 3.6 or greater
+eip712
+hexbytes
+web3.auto
```

Comment:
These could be added to the `requirement.txt` file. The version numbers can be fetched by running `pip freeze` from within the virtual environment this is running in.

Reply:
added